expertsystem/state/particle.py

```diff
@@ -40,6 +43,11 @@
 )
 
 
+def to_dict(input_ordered_dict: OrderedDict) -> dict:
+    """Convert nested `OrderedDict` to a nested `dict`."""
+    return loads(dumps(input_ordered_dict))
+
+
```

Comment:
Because this function is simply something needed for xml, I would plug this function into that scope.

Reply:
b89fbac754f5555de01412440c681dc84b721ea3